ansible/roles/test/tasks/wr_arp.yml

```diff
@@ -9,6 +9,10 @@
   script: roles/test/files/helpers/change_mac.sh
   delegate_to: "{{ ptf_host }}"
 
+- name: set ptf_host_ip
+  set_fact:
+    ptf_host_ip: "{{ hostvars[ptf_host]['ansible_host'] }}"
+
```

Comment:
why do we need this change?

Reply:
There are few places where we need to paste ptf_host IP address instead of ptf_host which actually contain the inventory hostname 
For example:
[Line 68](https://github.com/Azure/sonic-mgmt/pull/1354/files#diff-25615b83fc7f901c4b64a59270c2d545R68)
[Line 80](https://github.com/Azure/sonic-mgmt/pull/1354/files#diff-25615b83fc7f901c4b64a59270c2d545R80)

Reply:
ptf_host is used as ip_address with many places. Should we change it in other places too?

Reply:
I have done it in all places related to this test case. 
If you meant other places in the repo then, I guess yes, we should change it, but I did not search for such places. If you know them please open an issue or PR to fix.

Reply:
Previously we had ptf_host as ip address, now you're saying it must be a hostname. When was that changed?

Reply:
Some testbed specific changes cause this.

Reply:
@msosyak Have you investigated this?

Reply:
Yes. Basically ptf_host is set to ptf_ip [here](https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/test/tasks/sonic.yml#L70) .
But in many places mentioned that we should specify ptf_host env variable for specific tests and for some test as an example provided ip for another name of the container.

Also with new ansible to be able to delegate tasks to ptf it should be known for inventory. Even when we have ptf described in the inventory file like
`ptf_vms1-1    ansible_host=x.x.x.x    ansible_user=root  .....`
we should use the name of ptf host from inventory, not its ip address.

So here is the [PR](https://github.com/Azure/sonic-mgmt/pull/1370) with changes that should unify ptf host description and usage.


So, I think we should merge this PR as is.

Reply:
@msosyak Thank you for clarification. I can approve changes except related to ptf_host_ip. I don't know how the ptf_host works for now, so I can't approve it.